python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -76,6 +77,17 @@ class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
         doc="Weight exposures by airmass? Useful if there are relatively few high-airmass observations.",
         default=True,
     )
+    modelWidth = pexConfig.Field(
+        dtype=int,
+        doc="Width of the region around detected sources to include in the DcrModel."
+            "Set to a negative number to disable, which will include all pixels.",
```

Comment:
Unless you care to convince me otherwise, I'd prefer an error to be raised if a nonsensical modelWidth < 0 is chosen. Later it uses a negative value to set all the weights to 1, which seems dangerous; wouldn't throwing an exception be safer? It's also not clear to me why 3 is a good default.

Reply:
That's a good point. I'll make it raise an exception on negative values, and will add a separate config parameter to turn it on or off.